stock_split_picking/models/stock_picking.py

```diff
@@ -24,6 +24,19 @@ def _check_split_process(self):
                 )
             )
 
+        if all(
+            [
+                x.qty_done == x.product_uom_qty
```

Comment:
as `qty_done` and `product_uom_qty` are `float`, I would use the `float_compare` method from [odoo.tools](https://github.com/odoo/odoo/blob/14.0/odoo/tools/float_utils.py#L127)

Reply:
not blocking for this 😄